.github/integration/tests/sda/60_api_admin_test.sh

```diff
@@ -66,6 +66,42 @@ if [ -z "$output" ] ; then
     echo "Uploaded file not in inbox"
     exit 1
 fi
+
+# download the file, re-encrypted with the client key
+clientPubKey="$(base64 -w0 /shared/client.pub.pem)"
+outFile="download_reenc_NA12878.bam.c4gh"
```

Comment:
```suggestion
outFile="NC12878.bam.c4gh"
```

Reply:
Use another file at the later stage then or use another file ID here.
This test should use the -J -Oflags to ensure that the file is named correctly when the user doesn't supply an output filename.